app/namespace.go

```diff
@@ -1425,6 +1435,66 @@ func NewNamespaceCommand(getNamespaceClientFn GetNamespaceClientFn) (CommandOut,
 		},
 	})
 
+	// Note: the following commands are depend on previous commands. Please don't add more commands in the middle
+	exportCommand := subCommands[len(subCommands)-1]
+
+	// --- GCP Sink Related Commands ---
+	// Will move under export command once GCP sink is under public preview
+	if IsFeatureEnabled(GCPSinkFeatureFlag) {
+		exportCommand.Subcommands = append(exportCommand.Subcommands, &cli.Command{
+			Name:    "gcp-sink",
```

Comment:
so how does a user use this? `tcld es gcp validate?`

Reply:
also, whats the future facing approach for this?

we currently  have `tcld n es validate` 
is the right thing to move the current validate under a new validate function and have s3? e.g.

`tcld n es validate s3 <flags>` and then `tcld n es validate gcs <flags>`

Reply:
i think we should just make that above change now

Reply:
So I actually think another way of handling this. use `tcld n es s3 validate`